src/Menus/Menu.java

```diff
@@ -1,186 +1,48 @@
-
-                    if ("Utentes".equalsIgnoreCase(titulo)) {
-                        System.out.print("Introduza o nome: ");
-                        String nomeTemp = sc.nextLine();
-                        System.out.print("Introduza o NIF: ");
-                        String nifTemp = sc.nextLine();
-                        System.out.print("Introduza o género: ");
-                        String generoTemp = sc.nextLine();
-                        System.out.print("Introduza o contacto: ");
-                        String contactoTemp = sc.nextLine();
-
-                        utenteObj = new Utente(nomeTemp, nifTemp, generoTemp, contactoTemp);
-                        gerenciamentoUtentes.adicionarUtente(utenteObj);
-
-                    } else if ("Livros".equalsIgnoreCase(titulo)) {
-                        System.out.println("Introduza o Titulo: ");
-                        String tituloTemp = sc.nextLine();
-                        System.out.println("Introduza a editora: ");
-                        String editoraTemp = sc.nextLine();
-                        System.out.println("Introduza a Categoria: ");
-                        String categoriaTemp = sc.nextLine();
-                        System.out.println("Introduza o Ano de Edição: ");
-                        String anoDeEdicaoTemp = sc.nextLine();
-                        System.out.println("Introduza o ISBN: ");
-                        String isbnTemp = sc.nextLine();
-                        System.out.println("Introduza o Autor:");
-                        String autorTemp = sc.nextLine();
-
-                    } else {
-                        System.out.println("Função não implementada para " + titulo + ".");
-                    }
-                    break;
-
-                case 2:
-                    gerenciamentoUtentes.removerUtente();
-
-                    break;
-
-                case 3:
-                    System.out.println("Mostrar " + titulo + " selecionado.");
-                    gerenciamentoUtentes.listarUtentes();
-
-                    /*if (utenteObj != null && "Utentes".equalsIgnoreCase(titulo)) {
-                        System.out.println("Detalhes do Gerenciamento.Utente:");
-                        System.out.println("Nome: " + utenteObj.getNome());
-                        System.out.println("NIF: " + utenteObj.getNif());
-                        System.out.println("Gênero: " + utenteObj.getGenero());
-                        System.out.println("Contacto: " + utenteObj.getContacto());
-                    } else {
-                        System.out.println("Nenhum " + titulo + " foi adicionado ainda.");
-                    }*/
-                    break;
-                case 4:
-                    System.out.println("Alterar " + titulo + " selecionado.");
+    //mostra o menu
+    public void renderMenu() {
+        int x = 1;
+        Utils.Utils.printTituloPagina(title);
```

Comment:
podes sempre fazer no topo do ficheiro `import Utils.Utils;`

e depois utilizar ´so como 
```suggestion
       Utils.printTituloPagina(title);
```